<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2094</width>
    <height>1220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>40</y>
      <width>951</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color :qlineargradient(spread:pad, x1:0.631579, y1:0.222, x2:1, y2:1, stop:0 rgba(165, 81, 81, 255), stop:1 rgba(255, 255, 255, 255));
font-size: 30px;
}</string>
    </property>
    <property name="text">
     <string>Your answer: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>831</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color : green;
font-size: 30px;

}</string>
    </property>
    <property name="text">
     <string>write your question hier :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>831</width>
      <height>1031</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit{
font-size:30px;
}</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1810</x>
      <y>1120</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
background-color :rgb(174, 176, 155)
}</string>
    </property>
    <property name="text">
     <string>Developered by Ahmad Al Madani</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>200</y>
      <width>241</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color : green;
font-size: 30px;
}</string>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>70</y>
      <width>951</width>
      <height>1041</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser{
font-size: 30px;
}</string>
    </property>
    <property name="markdown">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>500</y>
      <width>241</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color :qlineargradient(spread:pad, x1:0.631579, y1:0.222, x2:1, y2:1, stop:0 rgba(165, 81, 81, 255), stop:1 rgba(255, 255, 255, 255));
font-size: 27px;

}</string>
    </property>
    <property name="text">
     <string>Clear output</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2094</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
